SharedSources/MediaLibraryModel/MediaModel.swift

```diff
@@ -48,16 +48,67 @@ extension VLCMLMedia {
         return ByteCountFormatter.string(fromByteCount: Int64(mainFile().size()),
                                          countStyle: .file)
     }
+}
+
+// convenience properties
+
+@objc extension VLCMLMedia {
```

Comment:
These properties could potentially live in the MediaLibrary

Reply:
Indeed, I'm not sure the app-side is the best place for all the properties.